recipes/openmesh/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+
+  export CC=clang
+  export CXX=clang++
+  export MACOSX_VERSION_MIN="10.7"
+  export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LDFLAGS="${LDFLAGS} -stdlib=libc++ -std=c++11"
+  export LINKFLAGS="${LDFLAGS}"
+  export MACOSX_DEPLOYMENT_TARGET=10.7
+else
+  SO_EXT='.so'
+fi
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_APPS=OFF \
+  -DOPENMESH_PYTHON_VERSION=${PY_VER} \
+  -DOPENMESH_BUILD_PYTHON_UNIT_TESTS=ON \
+  ..
+
+make install -j${CPU_COUNT} VERBOSE=1
```

Comment:
We have added support for `CPU_COUNT` going forward thanks to @jschueller's work in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/24 ). We now explicitly override this value on CIs to ensure it is set reasonably (after all CIs often misreport this). Feel free to make use of it in the future. If you are adding it to an existing feedstock, please re-render it locally with `conda-smithy` by doing the following.

1. `conda install -c conda-forge -n root conda-smithy`.
2. `cd <feedstock dir>` where `<feedstock dir>` is the repo cloned locally.
3. `conda smithy rerender`.
4. Commit and pushing all changes.